management/actions_test.go

```diff
@@ -47,6 +49,8 @@ func TestActionManager_Create(t *testing.T) {
 }
 
 func TestActionManager_Read(t *testing.T) {
+	setupVCR(t)
```

Comment:
Is there no way to pull this recorder setup function into a single point in the tests? I'm not sure if Go's native testing framework is comprehensive enough to support an init hook concept.

Reply:
Unfortunately there's no way for us to do that, as each test func runs in it's own separate routine and we depend on the name of the test case to generate the recordings. 

To achieve something like that we'd have to write our own test context that each test func extends from, but we'd still need to generate a separate file for each func, e.g. calling some func before running the givens, whens and the thens. Having it like this also gives us the benefit of immediately getting a 404 in case we forget to call this anywhere.